src/assets/pages/blog.json

```
@@ -1,5 +1,14 @@
+      "banner": "https://querido-diario-static.nyc3.cdn.digitaloceanspaces.com/blog/post0045/banner-post45.png",
+      "date": "05/15/2024",
+      "author": "Ester Lino (Brisa)"
```

Como você participou do QD nas Universidades como estudante da UNB é mais adequado deixar a sigla da instituição. 
```suggestion
      "author": "Ester Lino (UnB)"
```